src/Handler/CommandNameExtractor/NamedCommand.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace League\Tactician\Handler\CommandNameExtractor;
+
+/**
+ * Exposes a name for a command
+ */
+interface NamedCommand
+{
+    /**
+     * Returns the name of the command
+     *
+     * @return string
+     */
+    public function getName();
```

Comment:
`getName()` is a really common method on a command so we should change it to something less likely to collide with user params. `getCommandName()` seems pretty straightforward.